src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -435,6 +435,12 @@
     <Field Name="GetTypeFromHandle" Type="MethodSymbol" Null="allow"/>
   </Node>
 
+  <Node Name="BoundMethodToken" Base="BoundExpression">
```

Comment:
1) Yes. 2) Maybe, if lowering could be prevented from filling in GetMethodFromHandle.